client/core/helpers.go

```diff
@@ -310,29 +310,38 @@ func (ord *OrderReader) StatusString() string {
 
 // SimpleRateString is the formatted match rate.
 func (ord *OrderReader) SimpleRateString() string {
+	if ord.Type == order.MarketOrderType {
+		return ord.AverageRateString()
+	}
 	return ord.formatRate(ord.Rate)
 }
 
 // RateString is a formatted rate with units.
 func (ord *OrderReader) RateString() string {
+	rateStr := ord.formatRate(ord.Rate)
 	if ord.Type == order.MarketOrderType {
-		return "market"
+		rateStr = ord.AverageRateString()
 	}
```

Comment:
Is `"0"` better than `"market"` when there are no matches?

Reply:
Uhmm, I'm not too sure. But maybe we should say "market" if there are no matches.

Reply:
Or maybe just a "~".

Reply:
`"`market"` is better.

Reply:
> `"`market"` is better.

*IF* there are no orders, yh?